honeybee_energy/properties.py

```diff
@@ -0,0 +1,46 @@
+"""Energy Properties."""
+from .boundarycondition import BoundaryConditions
+
+
+class EnergyProperties(object):
+    """Energy Properties for Honeybee geometry."""
+
+    def __init__(self, srf_type):
+        self.construction = srf_type.energy_construction
+        self.boundary_condition = BoundaryConditions().outdoors
+
+    @property
+    def construction(self):
+        """Set or get Face construction."""
+        return self._construction
+
+    @construction.setter
+    def construction(self, value):
+        self._construction = value
+    
+    @property
+    def materials(self):
+        """List of materials."""
+        if not self._construction:
```

Comment:
Is it possible to initialize energy properties without a construction?

Reply:
It is possible. I have been thinking about the best way to implement it. I'm going to give it a try today and send a PR.

Reply:
Now it is initiated without a construction. The construction is set based on a generic construction-set and will be overwritten when faces are added to a zone unless they have been assigned by user. 

PS: I think I need to add zone to show what I mean here.

Reply:
This definitely feels like the right strategy.  I'm not so sure we want to have an `add_face` method on the zone unless we plan to implement the check to ensure that the zone is a closed volume in honeybee core but I agree with this general strategy and we can address this later.